com.sap.adt.abapcleaner/src/com/sap/adt/abapcleaner/programbase/CommandLineArgs.java

```diff
@@ -40,6 +42,9 @@ public static CommandLineArgs create(Persistency persistency, String[] args) {
 		final String LINE_SEP = System.lineSeparator();
 		
 		String sourceCode = null;
+		String[] sourceFiles = null;
```

Comment:
please change "sourceFiles" to "sourcePaths": In ABAP cleaner, "path" always means a full path "C:\any\dir\file.txt", while "file" could be a file "file.txt" without directory